cmd/slsa-provenance/cli/container.go

```diff
@@ -53,7 +53,7 @@ func OCI() *cobra.Command {
 				return err
 			}
 
-			opts := oci.WithDefaultClientOptions(cmd.Context(), true)
```

Comment:
originally we always took the MultKeychain approach. Now that has to be explicitly enabled commandline.